test/blackbox-tests/export-invalid-names.t

```diff
@@ -0,0 +1,10 @@
+Export otherwise invalid OCaml identifiers to JavaScript
+
+  $ . ./setup.sh
+  $ cat > x.ml <<EOF
+  > let [@mel.as POST] post= 1
+  > let [@mel.as "GET"] get = 2
+  > let [@mel.as "put"] other = 2
+  > EOF
+
+  $ melc -ppx melppx x.ml
```

Comment:
Maybe the test could check the output, for completion. Would be great to exercise the scenario with mlis and the one with JS reserved words as well.

Reply:
Oh I see, it's just that the snapshot [hasn't been updated](https://github.com/melange-re/melange/actions/runs/6061990027/job/16447763595?pr=714#step:6:1358).

I wonder about the reason to keep exporting both values? Wouldn't it be cleaner to export just the one in the payload?